src/editor/parts.ts

```diff
@@ -650,7 +651,9 @@ export class PartCreator {
         userId: string,
     ): [UserPillPart, PlainPart] {
         const pill = this.userPill(displayName, userId);
-        const postfix = this.plain(insertTrailingCharacter ? ": " : " ");
+        const postfix = this.plain(
+            insertTrailingCharacter && SettingsStore.getValue("MessageComposerInput.insertTrailingComma") ? ": " : " ",
```

Comment:
Because I'm paranoid:
```suggestion
            insertTrailingCharacter && (SettingsStore.getValue("MessageComposerInput.insertTrailingComma") ? ": " : " "),
```